client/src/components/client-area/OrderInProgress.vue

```diff
@@ -68,8 +109,9 @@
         <div
           class="details-list"
           v-if="
-            getStatus(customOrderBmsStatus, customOrderSellsyStatus)?.stepName === (StatusStep.Confirmed || StatusStep.Available) &&
-            orderDetails
+            orderDetails &&
+            [StatusStep.Confirmed, StatusStep.InvoiceToBePaid, StatusStep.Available].includes(
+              getStatus(customOrderBmsStatus, customOrderSellsyStatus)?.stepName ?? StatusStep.Unknown)
```

Comment:
You don't need the `?? StatusStep.Unknown`. `?.stepName` can be unknown, which will not be in the array.